presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -52,15 +53,18 @@
 {
     private final TTransport transport;
     private final ThriftHiveMetastore.Client client;
+    private final HostAndPort address;
```

Comment:
nit: Move this before the `transport` to maintain the same order of fields as in the constructor